README.md

```diff
@@ -22,6 +22,24 @@ docker compose run shell
 
 **Note:** `node_modules` are managed inside the container. You do not need to run `npm install` on your host.
 
+## Running Tests
+
+This project uses BDD (Behavior-Driven Development) tests with Gherkin syntax powered by Cucumber.js.
+
+### Running Unit Tests
```

Comment:
They aren't quite unit tests, as they do need the staging site to be up. I'd say integration tests?